src/combinatorics/simple_graph/strongly_regular.lean

```diff
@@ -0,0 +1,68 @@
+/-
+Copyright (c) 2021 Alena Gusakov. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Alena Gusakov
+-/
+import combinatorics.simple_graph.basic
+import data.set.finite
+/-!
+# Strongly regular graphs
+
+## Main definitions
+
+* `common_neighbors` is the intersection of the neighbor sets of two given vertices
+
```

Comment:
This should be moved to the module doc of `combinatorics.simple_graph.basic`, where `common_neighbors` is defined, right?

Reply:
Ah yeah, I'll move it